libcodechecker/libhandlers/store.py

```diff
@@ -223,15 +225,16 @@ def collect_file_hashes_from_plist(plist_file):
         for f in files:
             plist_file = os.path.join(input_path, f)
             if f.endswith(".plist"):
-                if collect_file_hashes_from_plist(plist_file):
+                missing_files = collect_file_hashes_from_plist(plist_file)
+                if not missing_files:
                     LOG.debug(
                         "Copying file '{0}' to ZIP assembly dir..."
                         .format(plist_file))
                     plist_report_files.append(os.path.join(input_path, f))
                 else:
-                    LOG.warning("Skipping '{0}' because it contains "
-                                "a missing source file."
-                                .format(plist_file))
+                    LOG.warning("Skipping '%s' because it contains "
```

Comment:
`contains` isn't the right word here from the user's perspective, and wasn't before, either...

Maybe `refers`?